ADR/0009-pipeline-service-via-operator.md

```diff
@@ -48,9 +48,7 @@ the OSP CRDs. The following changes are specific to RHTAP:
 Furthermore, as the service will be accessed through CodeReadyToolchain (CRT), the
 following changes are also specific to RHTAP:
 - Deploying a proxy (known as `SprayProxy`) on the CRT host cluster that redirects
```

Comment:
Do you think it'd be useful to have a link to the new ADR here?

Reply:
My feeling was these has to sort of self contained, but I saw such links in other ADRs, so added the link as suggested.